config.default.jsonc

```diff
@@ -1,26 +1,115 @@
 {
-  "decimals": 12,
-  "rateDifferencePercentThreshold": 25,
-  "port": 36661,
-  "refreshInterval": 10,
-  "minSources": 1,
+  "decimals": 12, // Number of decimal places for rate values
+  "rateDifferencePercentThreshold": 25, // Percentage threshold for notifying significant rate deviations
+  "refreshInterval": 10, // The frequency of rate updates from sources in minutes (optional). Frequent refreshes may require paid API keys
+  "minSources": 1, // Minimum number of sources required for rate calculation
```

Comment:
Let's describe exactly how updating a specific coin rate works. It worth creating a separate section in theWiki.

Maybe `"minSources": 2`?

Let's say we storing and providing rates for the ADM coin.

- If `"minSources": 2`, but ADM has the only source and it fetched successfully, will the new rate be stored?
- If ADM has 3 sources, but one of them failed, will the new rate be stored?
- If ADM has 3 sources: 0.03, 0.032, and 0.06, will the new rate be stored? Which one?
- If ADM has 3 sources: 0.03, failed, and 0.06, will the new rate be stored?
- If ADM rate wasn't stored, what will the `/get` endpoint return for `ADM/quote` pairs? Nothing (not included), null, or the previous value? If the previous value, what timeframe considered to be valid? What about the `/getHistory` endpoint?
- Describe `rateDifferencePercentThreshold` meaning
- Refer to this section from the config file

@martiliones @bludnic